test/cli/cli_install_static_test.go

```diff
@@ -1,46 +0,0 @@
-package test
```

Comment:
This file is used in the `windows_static_cli_tests` job in `./.github/workflows/release.yml`.